.circleci/config.yml

```diff
@@ -245,7 +245,7 @@ jobs:
               . activate habitat; cd habitat-lab
               python setup.py develop --all
               export PYTHONPATH=.:$PYTHONPATH
-              python setup.py test
+              python setup.py test --addopts "--cov-report=xml --cov=./"
```

Comment:
@Skylion007 Running codecov locally was really fighting me today so I made it just run on the CI.  I assume there was no reason it was running locally before?

Reply:
No, not really